### DISCUSSION TITLE
我跑到绳网应该没有村里人了吧

### DISCUSSION THREAD

**Discussion by MingSun-y** [share121/inter-knot#2164] [其他]
鸣隼:
不要沉迷女色，来继承火之意志吧！火之意志，格斗重燃，正宗格斗，激情燃烧，上线即送 SSS 火影！
![微信图片_20240815093504](https://github.com/user-attachments/assets/825d3d4f-d174-44f9-99db-1d4bc07e9089)

**Comment by jewlk** [comment]
包有的

**Comment by kirito621** [comment]
包的

**Comment by GWWA** [comment]
我勒个火批啊

**Comment by AAAEllenJoe** [comment]
包有的

**Comment by hws-twl** [comment]
包的

**Comment by ddddjoker** [comment]
包有的

**Comment by Gjxkxjaka** [comment]
包有的老弟
![Uploading emotion_download_1723723140337.jpeg…]()

**Comment by yours-an** [comment]
加纳

**Comment by yileinafs** [comment]
不是，死水还阴啊？

**Comment by God-Arch** [comment]
似水还音啊？

**Comment by caoling1423** [comment]
包的呀，小猫~老弟~

**Comment by Usodaru48763** [comment]
包的呀牢底

**Comment by M1nam1Yume** [comment]
包有的呀牢弟

**Comment by Brighttaz** [comment]
斯睡海因啊

**Comment by JobNumber-9527** [comment]
我勒个火之意志啊，我直接摸头起手<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/33b49a8bbead2a9e85290a75160d584e_2942391851204099012.png" alt="刃 来了" width="75" >
